<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="ipLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="conectarButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="desconectarButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Desconectar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>201</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="atualizarButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Atualizar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Timing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="comecarButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Come√ßar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pararButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetPlotter" native="true">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>29</y>
      <width>231</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
